libs/icons/scripts/optimize-svgs.ts

```diff
@@ -108,17 +134,18 @@ const getSvgs = async (srcSvgDir: string, distSvgDir: string, distSageIconsDir:
       );
 
       return {
+        distSvgFilePath,
+        exportName,
         fileName,
-        title,
+        fileNameCjs,
+        fileNameMjs,
+        iconName,
+        optimizedFilePath,
+        optimizedSvgContent:  isOptimizing ? null : await fs.readFile(srcFilePath, 'utf8'),
```

Comment:
This will be null if we are optimizing SVGs, otherwise, read what was already in our optimized folder (src/svg).